tests/foreman/cli/test_remoteexecution.py

```diff
@@ -1068,16 +1074,47 @@ def test_positive_run_reccuring_job(self):
 
         :expectedresults: multiple asserts along the code
 
-        :caseautomation: notautomated
+        :caseautomation: automated
 
         :CaseLevel: System
         """
+        # set connecting to host by ip
+        Host.set_parameter({
+            'host': self.client.hostname,
+            'name': 'remote_execution_connect_by_ip',
+            'value': 'True',
+        })
+        invocation_command = make_job_invocation({
+            'job-template': 'Run Command - Ansible Default',
+            'inputs': 'command="ls"',
+            'search-query': "name ~ {0}".format(self.client.hostname),
+            'cron-line': '* * * * *',  # every minute
+            'max-iteration': 2,  # just two runs
+        })
+        JobInvocation.get_output({
+            'id': invocation_command[u'id'],
+            'host': self.client.hostname
+        })
+        try:
+            self.assertEqual(invocation_command['status'], u'queued')
+        except AssertionError:
+            result = 'host output: {0}'.format(
+                ' '.join(JobInvocation.get_output({
+                    'id': invocation_command[u'id'],
+                    'host': self.client.hostname})
+                    )
+                )
+            raise AssertionError(result)
+        sleep(150)
```

Comment:
can it wait in a loop like this? https://github.com/SatelliteQE/robottelo/blob/a06898dc92569da6a126daf9eaf20b3593f867fe/tests/foreman/cli/test_remoteexecution.py#L430-L434
just to give it a chance to not take 2.5 min. every time